Testscripts/Windows/SETUP-RemoveIsoFromDvd.ps1

```diff
@@ -61,6 +61,31 @@ function Main {
         return $False
     }
 
+# Get Hyper-V VHD path
+$obj = Get-WmiObject -ComputerName $hvServer -Namespace "root\virtualization\v2" -Class "MsVM_VirtualSystemManagementServiceSettingData"
+$defaultVhdPath = $obj.DefaultVirtualHardDiskPath
+if (-not $defaultVhdPath) {
+    "Error: Unable to determine VhdDefaultPath on Hyper-V server ${hvServer}"
+    $error[0].Exception
+    return $False
+}
+if (-not $defaultVhdPath.EndsWith("\")) {
+    $defaultVhdPath += "\"
+}
+
+# Remove the .iso file
+$isoPath_tmp = $defaultVhdPath + "${vmName}_CDtest.iso"
+$isoPath = $isoPath_tmp.Replace(':','$')
+$isoPath = "\\" + $HvServer + "\" + $isoPath
+
+if (Test-Path "${isoPath}"){
+    try{
+        Remove-Item "${isoPath}"
```

Comment:
if (Test-Path "${isoPath}") { 
    try {

Reply:
Done, thank you